src/style.css

```diff
@@ -0,0 +1,113 @@
+@import "./stars.css";
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body,
+h2,
+p {
+  margin: 0;
+}
+
+:root {
+  --blue: #00acdc;
+  --main: #060b35;
+  --secondary: #616070;
+}
+
+.card {
+  padding-top: 32px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 198px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+}
+
+.card__content {
+  padding: 40px 16px 16px;
+}
+
+.card__product-name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--main);
+}
+
+.card__product-code {
+  margin-top: 4px;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--secondary);
+}
+
+.card__rating {
+  margin: 16px 0 24px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: flex-end;
+}
+
+.card__reviews {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--main);
+}
+
+.card__price {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: flex-end;
+  gap: 60px;
+}
+
+.card__price-text {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--secondary);
+}
+
+.card__price-price {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--main);
+}
+
+.card__buy {
+  margin-top: 16px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-decoration: none;
```

Comment:
Try to organize your styles
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)